client/common/cmdline.c

```diff
@@ -2506,12 +2528,36 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 
 			if (arg->Value)
 			{
-				if (_strnicmp("AVC444", arg->Value, 7) == 0)
+				int rc = CHANNEL_RC_OK;
+				char** p;
+				size_t count, x;
+
+				p = freerdp_command_line_parse_comma_separated_values(arg->Value, &count);
+				if (!p || (count == 0))
+					rc = COMMAND_LINE_ERROR;
+				for (x = 0; x < count; x++)
 				{
-					settings->GfxAVC444 = TRUE;
+					const char* val = p[x];
+
+					if (_strnicmp("AVC444", val, 7) == 0)
+					{
+						settings->GfxAVC444 = TRUE;
+					}
+					else if (_strnicmp("AVC420", val, 7) != 0)
+						rc = COMMAND_LINE_ERROR;
```

Comment:
I think you can exit the loop here.

Reply:
No, the arguments may appear in any order.

Reply:
I meant that you can exit in this case because rc = COMMAND_LINE_ERROR, so the whole operation will fail either way - there is no way to continue looping the modifing the settings struct